proxy_test.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+
+	"github.com/cloudflare/unsee/internal/alertmanager"
+
+	httpmock "gopkg.in/jarcoal/httpmock.v1"
+)
+
+// httptest.NewRecorder() doesn't implement http.CloseNotifier
+type closeNotifyingRecorder struct {
```

Comment:
I don't see you using the CloseNotify method. Is this recorder necessary?

Reply:
I'm not 100% sure as I didn't go that deep into the issue, but testes were failing because of it and after some research I wasn't sure if this is gin issue or httptest

Reply:
I don't have time to research myself, but if you eventually figure it out, I'd like to know.

Reply:
Is that blocking this merge?

Reply:
No.